### PR TITLE
Added Void Linux installation instructions

### PR DESCRIPTION
**Pull request by notramo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,10 @@ text_font=sans-serif
 - [Arch Linux (git)](https://aur.archlinux.org/packages/swappy-git)
 - [Fedora 31/32](https://copr.fedorainfracloud.org/coprs/wef/swappy)
 - [openSUSE](https://build.opensuse.org/package/show/X11:Wayland/swappy)
+- Void Linux
+  ```bash
+  sudo xbps-install swappy
+  ```
```
Comment (jtheoof) [inline README.md:104]: Can you simply point to the package like it's done above ? It's up the to distro users to know how to install those packages.
Also the commit is not respecting `conventional-changelog`.

Comment (notramo) [reply README.md:104]: The [online package catalog for Void Linux](https://voidlinux.org/packages/) doesn't include the package name in the URL.
Also, Void Linux doesn't have much package managers, like Arch Linux, just CLI XBPS, and a GUI for it (that nobody uses).
Would it be better to have this?
> Void Linux: the `swappy` package in the repositories.

Comment (jtheoof) [reply README.md:104]: Sure sounds good enough

**Comment by jtheoof** [comment]
@notramo friendly ping to update the code and commit please.

**Comment by notramo** [comment]
I tried to search for conventional-changelog but I didn't find instructions to it.

**Comment by notramo** [comment]
Couldn't force-push from command line, and don't know how to do it from web. I will open another PR.

**Pull request closed without merging**